### ISSUE TITLE
rxPaginate: combining rxBulkSelect and rxLoadingOverlay in same directive causes angular error

### ISSUE THREAD

**Issue by ErnieAtLYD** [rackerlabs/encore-ui#1622]
- **Details about your browser and operating environment:** Verified on both Firefox and Chrome.
 - **Framework version:** 1.45.2
 - **Perspective on whether this issue is blocking your team (if relevant), or how it could improve EncoreUI usage (this helps the Framework team prioritize your issue)**: Flagged during a code review. Right now I can get around this by using a bunch of `<rxNotify>` tags around promises. If this is something that won't be fixed for a while, updating the docs would be a good idea.
 - **Relevant Pull Request**: DCX UCP, https://github.rackspace.com/dcx/ucp/pull/211

HTML:
```
    <table
      rx-bulk-select
      rx-floating-header
      rx-loading-overlay
      bulk-source="pager"
      resource-name="device"
      selected-key="rowIsSelected"
      class="table-striped">  
```

Angular Error:
```
Error: [$compile:multidir] Multiple directives [rxBulkSelect, rxLoadingOverlay] asking for new/isolated scope on: <table rx-bulk-select="" rx-floating-header="" rx-loading-overlay="" bulk-source="pager" resource-name="device" selected-key="rowIsSelected" class="table-striped">
http://errors.angularjs.org/1.2.29/$compile/multidir?p0=rxBulkSelect&p1=rxLoadingOverlay&p2=new%2Fisolated<ng-view class="ng-scope">cope&p3=%3Ctable%20rx-bulk-select%3D%22%22%20rx-floating-header%3D%22%22%20rx-loading-overlay%3D%22%22%20bulk-source%3D%22pager%22%20resource-name%3D%device%22%20selected-key%3D%22rowIsSelected%22%20class%3D%22table-striped%22%3E
    at http://127.0.0.1:9000/dcx/control-panel-encore/bower_components/angular/angular.js:78:12
    at assertNoDuplicate (http://127.0.0.1:9000/dcx/control-panel-encore/bower_components/angular/angular.js:7009:15)
    at applyDirectivesToNode (http://127.0.0.1:9000/dcx/control-panel-encore/bower_components/angular/angular.js:6404:13)
    at compileNodes (http://127.0.0.1:9000/dcx/control-panel-encore/bower_components/angular/angular.js:6093:15)
    at compileNodes (http://127.0.0.1:9000/dcx/control-panel-encore/bower_components/angular/angular.js:6105:15)
    at compile (http://127.0.0.1:9000/dcx/control-panel-encore/bower_components/angular/angular.js:6026:15)
    at applyDirectivesToNode (http://127.0.0.1:9000/dcx/control-panel-encore/bower_components/angular/angular.js:6457:33)
    at compileNodes (http://127.0.0.1:9000/dcx/control-panel-encore/bower_components/angular/angular.js:6093:15)
    at compile (http://127.0.0.1:9000/dcx/control-panel-encore/bower_components/angular/angular.js:6026:15)
    at link (http://127.0.0.1:9000/dcx/control-panel-encore/bower_components/angular-route/angular-route.js:903:18)(anonymous function) @ angular.js:10147(anonymous function) @ angular.js:7414nodeLinkFn @ angular.js:6771compositeLinkFn @ angular.js:6155publicLinkFn @ angular.js:6051boundTranscludeFn @ angular.js:6175controllersBoundTransclude @ angular.js:6789update @ angular-route.js:865Scope.$broadcast @ angular.js:13133(anonymous function) @ angular-route.js:547wrappedCallback @ angular.js:11722wrappedCallback @ angular.js:11722(anonymous function) @ angular.js:11808Scope.$eval @ angular.js:12851Scope.$digest @ angular.js:12663Scope.$apply @ angular.js:12955(anonymous function) @ angular.js:1449invoke @ angular.js:3967doBootstrap @ angular.js:1447bootstrap @ angular.js:1461angularInit @ angular.js:1370(anonymous function) @ angular.js:22164fire @ jquery.js:3099self.fireWith @ jquery.js:3211jQuery.extend.ready @ jquery.js:3417completed @ jquery.js:3433
```

**Comment by CITguy** [comment]
Will placing `rx-loading-overlay` on a div wrapping the table still result in the same error?

```
<div rx-loading-overlay>
  <table
      rx-bulk-select
      rx-floating-header
      bulk-source="pager"
      resource-name="device"
      selected-key="rowIsSelected"
      class="table-striped">  
   ...
  </table>
</div>
```

**Comment by ErnieAtLYD** [comment]
Oh hey, it does. But how do I call the `hide` and `show` method manually? My UI requires that table views starts at a neutral state (0 items listed, no API is called), and for some reason that prevents the loading screen from going away.

**Comment by 100stacks** [comment]
@ErnieAtLYD, https://github.com/rackerlabs/encore-ui/pull/1623 resolved this issue.

**Comment by CITguy** [comment]
Corrected in (`encore-ui 1.46.3`](https://github.com/rackerlabs/encore-ui/releases/tag/v1.46.3)
